### PR TITLE
Add tests for searching works

### PR DESCRIPTION
**Pull request by Elarnon**

### CODE REVIEW

File: mangaki/mangaki/tests/test_work.py [hunk 1]
```diff
@@ -61,6 +62,32 @@ def setUp(self):
...
+            ('boxers', None),
+        ]
+        for query, expected in tests:
+            qs = list(Work.objects.filter(title__search=query))
+            if expected is None:
+                self.assertFalse(qs)
+                continue
+            elif isinstance(expected, (list, tuple)):
+                self.assertEqual(len(qs), len(expected))
+                for w in qs:
```
Comment (RaitoBezarius) [inline mangaki/mangaki/tests/test_work.py]:
```python
for w, e in zip(qs, expected):
    self.assertIn(w, expected)
    self.assertIn(e, qs)
```

?

**Comment by RaitoBezarius** [comment]
Please rebase :+1: !

File: mangaki/mangaki/tests/test_work.py (hunk 1 above)
Comment (Elarnon) [inline mangaki/mangaki/tests/test_work.py]: I prefer not to introduce possible weird cases with zip & different length (even though right now we ensure that `qs` and `expected` have the same length). There is value in keeping tests as simple as possible.

Comment (RaitoBezarius) [inline mangaki/mangaki/tests/test_work.py]: You have a point, let's do that!

**Pull request merged**